website/docs/r/appstream_fleet.html.markdown

```diff
@@ -0,0 +1,99 @@
+---
+subcategory: "AppStream"
+layout: "aws"
+page_title: "AWS: aws_appstream_fleet"
+description: |-
+  Provides an AppStream fleet
+---
+
+# Resource: aws_appstream_fleet
+
+Provides an AppStream fleet.
+
+## Example Usage
+
+```terraform
+resource "aws_appstream_fleet" "test_fleet" {
+  name = "test-fleet"
+  compute_capacity {
+    desired_instances = 1
+  }
+  description                        = "test fleet"
+  idle_disconnect_timeout_in_seconds = 15
+  display_name                       = "test-fleet"
+  enable_default_internet_access     = false
+  fleet_type                         = "ON_DEMAND"
+  image_name                         = "Amazon-AppStream2-Sample-Image-02-04-2019"
+  instance_type                      = "stream.standard.large"
+  max_user_duration_in_seconds       = 600
+  vpc_config {
+    subnet_ids = ["subnet-06e9b13400c225127"]
+  }
+  tags = {
+    TagName = "tag-value"
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) A unique name for the fleet.
+* `name_prefix` -  (Optional) Creates a unique name beginning with the specified prefix. Conflicts with `name`.
+* `compute_capacity` - (Required) The desired capacity for the fleet. (documented below)
+* `description` - (Optional) The description to display.
+* `disconnect_timeout_in_seconds` - (Optional) The amount of time that a streaming session remains active after users disconnect.
+* `display_name` - (Optional) Human-readable friendly name for the AppStream fleet.
+* `domain_join_info` - (Optional) The name of the directory and organizational unit (OU) to use to join the fleet to a Microsoft Active Directory domain. (documented below)
+* `enable_default_internet_access` - (Optional) Enables or disables default internet access for the fleet.
+* `fleet_type` - (Optional) The fleet type. Valid values are: `ON_DEMAND`, `ALWAYS_ON`
+* `iam_role_arn` - (Optional) The Amazon Resource Name (ARN) of the IAM role to apply to the fleet.
```

Comment:
```suggestion
* `iam_role_arn` - (Optional) ARN of the IAM role to apply to the fleet.
```
* Never begin the description with `The ` or `A `. Please fix all the arguments and attributes.
* ARN should never be spelled out - just `ARN` is sufficient.